docs/vaahcms-2/backend/api-authentication.md

```diff
@@ -402,12 +402,12 @@ Create new user with HTTP request. During the creation you can set up attributes
 
 ##### URL
 
-```
+```php
 GET <public-url>/api/users/create?api_token=xxxxxxxxxxx
```

Comment:
@wri-abinash-s001 `api_token` should not be sent in url, instead it should be sent in the header. 

example for code of `javascript`:

```javascript

let config = {
    headers: {
        'Accept': 'application/json',
        'Authorization': `Bearer ${api_token}`
    }
};

let response = await this.axios.get("/example-api-url", config);
```